linera-views/src/batch.rs

```diff
@@ -82,7 +82,7 @@ pub struct UnorderedBatch {
 }
 
 impl UnorderedBatch {
-    /// From an `UnorderedBatch`, creates a [`SimpleUnorderedBatch`] that does not contain the
+    /// From an `UnorderedBatch`, create a [`SimpleUnorderedBatch`] that does not contain the
```

Comment:
That was correct before. I'm not sure why Grammarly complains about it.

Reply:
I am inclined to think that Grammarly is correct as it is not really a sentence. 
For what is worth, https://quillbot.com/ has the same decision as Grammarly and https://www.online-spellcheck.com/ recognize both as correct.

Reply:
@MathieuDutSik I kinda prefer Grammarly's suggestion but we just updated the rest of the codebase in the other direction, so we're going to have to stick to it (unless the team wants to switch back at some point).

Reply:
Ok, for me, it was the imperative tense in English.
So, let's do as in the rest of the code.